### PR TITLE
Update README file: add Description, Installation, Authors and License sections.

### PR DESCRIPTION
**Pull request by pingvin1977**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# ios-school-homeworks
\ No newline at end of file
+# iOS School Homeworks
+
+[![CocoaPods Platforms](https://img.shields.io/cocoapods/p/m)](https://img.shields.io/cocoapods/p/m?color=gree)
```
Comment (dersim-davaod) [inline README.md:3]: 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+
+This repository contains practical homework solutions for learning the program language `swift`
```
Comment (dersim-davaod) [inline README.md:5]: I didn't understand the sentence:

> for learning the program language `swift` school "iOS CocoaHeads August 2020"

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
-# ios-school-homeworks
\ No newline at end of file
+# iOS School Homeworks
+
+[![CocoaPods Platforms](https://img.shields.io/cocoapods/p/m)](https://img.shields.io/cocoapods/p/m?color=gree)
+
+This repository contains practical homework solutions for learning the program language `swift` 
+school "iOS CocoaHeads August 2020". Homeworks is divided into directories.
```
Comment (dersim-davaod) [inline README.md:6]:
```suggestion
school "iOS CocoaHeads August 2020". Each homework is in the separate directory.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,18 @@
...
+
+## Installation
+You need [XCode](apps.apple.com/us/app/xcode/id497799835?mt=12) application to run jobs.
```
Comment (dersim-davaod) [inline README.md:9]: which jobs?

Comment (dersim-davaod) [inline README.md:9]: Please remember how to write IDE's name: not `XCode`, but `Xcode`.

File: README.md [hunk 5]
```diff
@@ -1 +1,18 @@
...
+In your chosen homework, you need to run file `contents.xcplayground` in the selected directory.
+
+## Autors
```
Comment (dersim-davaod) [inline README.md:12]:
```suggestion
## Authors
```

File: README.md [hunk 6]
```diff
@@ -1 +1,18 @@
...
+Thanks to my mentor [Dersim Davaod](github.com/dersim-davaod) who contributed a lot to this project and the volunteer
```
Comment (dersim-davaod) [inline README.md:13]: Please don't mention me :)
Also you don't have to mention ByChange (but you can if you want).

In the simplest form just leave it as:
```
## Authors
[Yury Gudyma](github-nickname)
```

as you're a main author.

File: README.md [hunk 7]
```diff
@@ -1 +1,18 @@
...
+group [ByChange](bychange.me) who made this project possible.
+
+
```
Comment (dersim-davaod) [inline README.md:16]:
```suggestion
```

**Review by dersim-davaod — CHANGES_REQUESTED** [review]
Consider the changes below.

Also please give your PR more descriptive name. E.g.:
Update README file: add Description, Installation, Authors and License sections.

Also there's a typo in the name of the PR: Autors -> Authors.

File: README.md (hunk 3 above)
Comment (pingvin1977) [reply README.md:6]: Fixed.

File: README.md (hunk 4 above)
Comment (pingvin1977) [reply README.md:9]: Fixed.

File: README.md (hunk 5 above)
Comment (pingvin1977) [reply README.md:12]: Fixed.

File: README.md (hunk 6 above)
Comment (pingvin1977) [reply README.md:13]: Changed the author.

File: README.md (hunk 7 above)
Comment (pingvin1977) [reply README.md:16]: Fixed.

File: README.md (hunk 4 above)
Comment (pingvin1977) [reply README.md:9]: Jobs corrected to works

File: README.md (hunk 2 above)
Comment (pingvin1977) [reply README.md:5]: Rewrote description.

File: README.md [hunk 8]
```diff
@@ -1 +1,16 @@
-# ios-school-homeworks
\ No newline at end of file
+# iOS School Homeworks
+
+[![CocoaPods Platforms](https://img.shields.io/cocoapods/p/m)](https://img.shields.io/cocoapods/p/m?color=gree)
+
+This repository contains completed homework tasks for learning the programming language `swift` 
+at the online school "iOS CocoaHeads August 2020". Each homework is in the separate directorys.
```
Comment (dersim-davaod) [inline README.md:6]:
```suggestion
at the online school "iOS CocoaHeads August 2020". Each homework is in the separate directory.
```

File: README.md [hunk 9]
```diff
@@ -1 +1,16 @@
...
+
+## Installation
+You need [Xcode](apps.apple.com/us/app/xcode/id497799835?mt=12) application to run works.
```
Comment (dersim-davaod) [inline README.md:9]: > _to run works_

Which works?

Comment (dersim-davaod) [reply README.md:9]:
```suggestion
The code is written in Xcode, so you need [Xcode](apps.apple.com/us/app/xcode/id497799835?mt=12) and CLT to run playgrounds/projects.
```

Comment (dersim-davaod) [reply README.md:9]: Also: which version of Xcode do I need? Is that ok to use Xcode 10?

**Review by dersim-davaod — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (pingvin1977) [reply README.md:6]: Fixed.

File: README.md (hunk 9 above)
Comment (pingvin1977) [reply README.md:9]: Added the Xcode version to the description and rewrote the installation.

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Pull request merged**